tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
So `rd2html` and friends fail to parse `\link[=label]{function}`. It's a separate (and more complicated) bug from the flip in argument order #194 that motivated this PR. Do you want me to file a separate issue and make a new PR on top of this one when I've figured it out or just keep going?